src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -1232,6 +1278,44 @@ class spell_warr_vigilance_trigger : public SpellScriptLoader
         }
 };
 
+// 1680 - Whirlwind 
+// Updated 7.1.5
+class spell_warr_whirlwind : public SpellScriptLoader
```

Comment:
Duplicate, Trauma are a proc so need only check condition for proc (whirlwind & slam) after your script `spell_warr_trauma` makes the rest